packages/story-editor/src/components/colorPicker/customColorPicker.js

```diff
@@ -80,6 +88,21 @@ function CustomColorPicker({
     }
   }, [color, load]);
 
+  const handleColorPickerChange = useCallback(
+    (e) => {
+      updateCurrentColor(e);
+      // If using the eyedropper in floating menu, the popup unmounts
+      // so the generatedColor won't ever be updated
+      // check for unmount and if so, we know this change event
+      // if from the eyedropper so we can just grab the rgb and
+      // trigger the rest of the change for the element.
+      if (!isMounted.current && e?.rgb) {
+        handleColorChange({ color: e.rgb });
+      }
+    },
+    [updateCurrentColor]
```

Comment:
This is missing a dependency
```suggestion
    [handleColorChange, updateCurrentColor]
```

Reply:
I really need to get my linter to start working again, thanks 🙈